composer.json

```diff
@@ -20,14 +20,15 @@
         "drift/http-kernel": "0.1.*, >=0.1.7",
         "drift/server": "0.1.*, >=0.1.7",
         "drift/react-functions": "0.1.*",
-        "drift/event-loop-utils": "0.1.*"
+        "drift/event-loop-utils": "0.1.*",
+        "drift/dbal-bundle": "^0.1.2"
```

Comment:
You need to add the mysql-bundle to the composer require block:

```
        "drift/dbal-bundle": "^0.1.2",
        "drift/mysql-bundle": "^0.1.1"
```

Reply:
Ok, I test that locally and update the PR